package.json

```diff
@@ -7,7 +7,9 @@
     "build": "tsc && cp -r views dist/",
     "devstart": "nodemon",
     "test": "jest --forceExit",
-    "lint": "tslint -p ."
+    "lint": "tslint -p .",
+    "sonarqube": "npm run coverage-jest && sonar-scanner",
+    "coverage-jest": "jest -c jest.config.coverage.js --forceExit --coverage"
```

Comment:
Target could be just 'coverage', for consistency with the 'test' target above, which also uses Jest but doesn't mention it in the target name.